seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/sink/writer/JsonWriteStrategy.java

```diff
@@ -103,6 +103,8 @@ public void finishAndCloseFile() {
                     }
                     needMoveFiles.put(key, getTargetLocation(key));
                 });
+        beingWrittenOutputStream.clear();
```

Comment:
What is the connection between this modification and timeout?

Reply:
no connection

Reply:
The beingWrittenOutputStream collection needs to be cleared at each checkpoint.

Reply:
good